### ISSUE TITLE
License info and repository info in package.json

### ISSUE THREAD

**Issue by zuphilip** [zotero/translation-server-v2#18]
Currently, I see the following warnings when installing it:
```
$ npm install
npm WARN translation-server@0.0.1 No repository field.
npm WARN translation-server@0.0.1 license should be a valid SPDX license expression
```

These two things should be easy to fix. For the license you may want to choose either `AGPL-3.0-only` or `AGPL-3.0-or-later`, see https://spdx.org/licenses/. For the repository field see https://docs.npmjs.com/files/package.json#repository.

**Comment by adomasven** [comment]
Closed in 776991c5dd82ced3259f063d8063e50d49f11f31

**Comment by zuphilip** [comment]
Thank you!
